cluster-autoscaler/simulator/cluster.go

```diff
@@ -29,6 +30,14 @@ import (
 	"github.com/golang/glog"
 )
 
+var (
```

Comment:
But then we'd have to push through all of the classes/methods. If follow the same logic should also have all the constants in the main file?

I think it's much cleaner if we have in modules that they actually control.